README.md

```diff
@@ -1 +1,36 @@
-# to-do-list
\ No newline at end of file
+# To-do list
+> This is a single page aplication for adding tasks and shows a list of all of them.
+
+You can add and remove tasks from the to-do list.
+
+## Built With
+
+- HTML
+- CSS
+- ES6
+- Webpack
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
```

Comment:
- [ ] Add instructions about how to run your project locally (e.g. npm install, npm start, etc)